3rd Party Guide for Altibase/eng/Spring Data JPA With Hibernate 6.4 User's Guide for Altibase.md

```diff
@@ -0,0 +1,235 @@
+
+<br/>
+
+# Overview
+
+This manual explains how to integrate Spring Data JPA with Hibernate 6.4 and an Altibase server using Spring Boot.
+
+The document is based on the following versions:
+
+- Spring Boot 3.2.2
+- Spring Data JPA 3.2.2
+- Java 17
+- Altibase server 7.1.0.9.3 or above
+- Altibase JDBC driver 7.1.0.9.0 or above
+
+<br/>
+
+# Integrate Altibase With Spring Boot
+
+## 1. Generate Spring Boot Project With Spring initializr
+
+Visit [https://start.spring.io/](#https://start.spring.io/), then enter the following information. After that, click on "Generate" to create the project. Click on "ADD DEPENDENCIES..." under Dependencies, and add "Spring Web" and "Spring Data JPA". For the convenience of writing example code, also add "Lombok".
+
+<img src="Images/JPA/spring_initializr.png"/>
+
+## 2. Add Dependencies to pom.xml
+
+### Add AltibaseDialect Dependency
+
+Starting from Hibernate 6.4, the AltibaseDialect has been added to hibernate-community-dialect. Therefore, add a dependency on "hibernate-community-dialect" with a version of 6.4 or above. (If users are using Hibernate version under 6.4, users need to manually compile and add the AltibaseDialect.)
+
+```xml
+<dependency>
+    <groupId>org.hibernate.orm</groupId>
+    <artifactId>hibernate-community-dialects</artifactId>
+    <version>6.4.1.Final</version>
+</dependency>
+```
+
+### Add Altibase JDBC Driver Dependency
+
+From patch version Altibase 7.1.0.9.0 for Altibase 7.1 and Altibase 7.3.0.0.2 for Altibase 7.3, the Altibase JDBC driver can be downloaded from the [Maven Central Repository](https://mvnrepository.com/artifact/com.altibase/altibase-jdbc). Add to Altibase JDBC driver dependency as follows: 
+
+* Add Altibase JDBC driver dependency for Altibase 7.3
+
+  ```xml
+  <dependency>
+      <groupId>com.altibase</groupId>
+      <artifactId>altibase-jdbc</artifactId>
+      <version>7.3.0.0.2</version>
+  </dependency>
+  ```
+
+* Add Altibase JDBC driver dependency for Altibase 7.1
+
+  ```xml
+  <dependency>
+      <groupId>com.altibase</groupId>
+      <artifactId>altibase-jdbc</artifactId>
+      <version>7.1.0.9.2</version>
+  </dependency>
+  ```
+
+## 3. Configure the application.properties
+
+### Altibase JDBC Driver Configuration
+
+```java
+# Database
+spring.datasource.driver-class-name=Altibase.jdbc.driver.AltibaseDriver
+spring.datasource.url=jdbc:Altibase://localhost:20300/mydb
+spring.datasource.username=sys
+spring.datasource.password=manager
+
+# JPA
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+```
+
+> [!NOTE]
+>
+> **spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true** 
+>
+> This configuration is to suppress the java.sql.SQLFeatureNotSupportedException error that may occur when Hibernate calls Connection.createNClob() to check if the NClob type is supported or not. Altibase does not support the NClob type, which leads to the occurrence of this error. However, it can be safely ignored.
```

Comment:
```suggestion
> This configuration is to suppress the java.sql.SQLFeatureNotSupportedException error that may occur when Hibernate calls Connection.createNClob() to check whether or not the NClob type is supported. Altibase does not support the NClob type, which causes this error to occur. However, it can be safely ignored.
```